packages/ui/src/components/Catalog/BaseCatalog.tsx

```diff
@@ -28,25 +30,31 @@ export const BaseCatalog: FunctionComponent<BaseCatalogProps> = (props) => {
     [onTileClick, props.tiles],
   );
 
+  useEffect(() => {
+    catalogBodyRef.current!.scrollTop = 0;
```

Comment:
I think in this case, we can leave it because the `div` is static, we just need to be aware that this might not be the case always 👀 and we can go 💥